modules/ROOT/pages/teams-example.adoc

```diff
@@ -0,0 +1,39 @@
+= Example Teams Structure
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Imagine that your organization, Acme, includes a team called Engineering that contains smaller teams that focus on building specific components. The Identity and DevOps teams are specialized teams within the larger Engineering team.
+
+Using the Teams feature, you can structure your users and permissions in the following way:
+
+image::access-management-teams-hierarchy.png[]
+
+The user teams shown here are defined as follows:
+
+* Acme +
+This team is your organization and parent team, and every user in your organization is a member. By default, the team’s name is Everyone at Acme. You can set your permissions such that all Acme users can easily collaborate within your organization by creating new projects in Anypoint Design Center and viewing assets in Anypoint Exchange. To do this, you must configure the following permissions for the Everyone at Acme team:
```

Comment:
```suggestion
This team is your organization and root team, and every user in your organization is a member. By default, the team’s name is Everyone at Acme. You can set your permissions such that all Acme users can easily collaborate within your organization by creating new projects in Anypoint Design Center and viewing assets in Anypoint Exchange. To do this, you must configure the following permissions for the Everyone at Acme team:
```

Reply:
I don't like the phrasing of `this team is your organization` - teams aren't organizations